### ISSUE TITLE
[new sysrepo] Cannot create session on i386

### ISSUE THREAD

**Issue by gitter-runner** [sysrepo/sysrepo#1748]
Hi,

I get an error when calling _sr_session_start()_ on a 32-bit sysrepo build (Ubuntu 18.04/i386):

`[ERR]: Condition variable address not aligned.`

Could you look into this?

sysrepo version: 1.3.41
libyang version: 1.0.118

Thanks!

**Comment by michalvasko** [comment]
Hi,
where exactly did you create the session (a callback, your applicaiton `main`, ...)? Have you tried running tests, did they pass?

Regards,
Michal

**Comment by gitter-runner** [comment]
Hi,

the first _sr_session_start()_ fails in all applications, right after creating sysrepo connection.
Sysrepocfg also fails with the same error when I try to import configuration data.
I attached a docker image to reproduce the former sysrepocfg error.

```
[ERR]: Condition variable address not aligned.
sysrepocfg error: Failed to start a session (Internal error)
[ERR]: Locking a mutex failed (sr_ptr_del: Invalid argument).
Segmentation fault (core dumped)
```

[sysrepo-alignment-issue.zip](https://github.com/sysrepo/sysrepo/files/4059508/sysrepo-alignment-issue.zip)

Thanks!

**Comment by michalvasko** [comment]
Hi,
this was already fixed (a201e92a4), please use `devel` for now.

Regards,
Michal
